packages/fetchai/connections/p2p_libp2p/aea/api.go

```diff
@@ -51,6 +51,15 @@ type Pipe interface {
 	Close() error
 }
 
+// Needed to break import cycle
```

Comment:
For now I am using this type only in `aea`. Elsewhere I am using the protobuf message directly